cpp/CMakeLists.txt

```diff
@@ -220,6 +231,15 @@ endif()
 rapids_cpm_init()
 rapids_cmake_install_lib_dir(lib_dir)
 
+# Not using rapids-cmake since we never want to find, always download.
+CPMAddPackage(
+    NAME rapids_logger GITHUB_REPOSITORY rapidsai/rapids-logger GIT_SHALLOW FALSE GIT_TAG
+    4df3ee70c6746fd1b6c0dc14209dae2e2d4378c6 VERSION 4df3ee70c6746fd1b6c0dc14209dae2e2d4378c6
+)
+rapids_make_logger(
+  ML EXPORT_SET cuml-exports LOGGER_HEADER_DIR include/cuml/common/ LOGGER_MACRO_PREFIX CUML LOGGER_TARGET cuml_logger
+)
```

Comment:
I presume this is something that happens in all repos that use rapids-logger, I wonder if a function in rapids-cmake would be a good idea instead of using CPMAddPackage directly?

Reply:
Yes absolutely. Every repository will have to do its own call to `rapids_make_logger` to provide the right arguments, but I plan to replace the `CPMAddPackage` call with a rapids-cmake call. I will do that a bit later though once I'm ready to synchronize all the repos because right now they are using different commit hashes and I'll be adding a couple more features to the trunk of rapids-logger before I synchronize all the repos using it.

Reply:
Starting to address this in https://github.com/rapidsai/rapids-cmake/pull/737.